### DISCUSSION TITLE
MARCAuth 5XX relationships in LC names/ resources, rwo/ resources

### DISCUSSION THREAD

**Discussion by briesenberg07** [uwlib-cams/sinopia_maps#166] [seeking feedback]
## Proposals
- **Do not** add LC Names lookups, which yield resources typed as [madsrdf:Authority](http://www.loc.gov/mads/rdf/v1#Authority), to property templates which are for creating relationships to non-WEMI resources such as persons, corporate bodies, families, or places.
- **Provide training** for Sinopia catalogers on obtaining LC RWO resource IRIs using the browser where such resources cannot be searched using a lookup from within Sinopia. 

## Details
The lookups currently configured for the property template 'has related corporate body of work' include (along with some other sources) the following LC lookup sources:

- LOC person [names (rwo)] (QA)
- LOC organization [names (rwo)] (QA)
- LOC family [names (rwo)] (QA)

These lookup sources yield resources typed as [madsrdf:RWO](http://www.loc.gov/mads/rdf/v1#RWO):

> Label: Real World Object
> Comment: A madsrdf:RWO is an abstract entity and identifies a Real World Object (RWO) identified by the label of a madsrdf:Authority or madsrdf:DeprecatedAuthority.

Such resources are often assigned additional types:
- For persons - [bf:Person](http://id.loc.gov/ontologies/bibframe/Person) and [foaf:Person](http://xmlns.com/foaf/0.1/#term_Person)
- For organizations - [bf:Organization](http://id.loc.gov/ontologies/bibframe/Organization) and [foaf:Organization](http://xmlns.com/foaf/0.1/Organization)
- For families - [bf:Family](http://id.loc.gov/ontologies/bibframe/Family) and [foaf:Group](http://xmlns.com/foaf/0.1/Group)

Adam S. requested the addition of two lookup sources, to the 'has related corporate body of work' property template, one of these is the [LOC geographic [names] (QA)](https://github.com/LD4P/sinopia_editor/blob/f7d3a6f064f40d612de7c03ddc8491f9bdda9aaa/static/authorityConfig.json#L1102-L1109) lookup. This lookup yields resources are typed as [madsrdf:Authority](http://www.loc.gov/mads/rdf/v1#Authority):

> Label: Authority
> Comment: A concept with a controlled label.

The argument for adding an LC names lookup here is the desire to select value resources typed as  [madsrdf:Geographic](http://www.loc.gov/standards/mads/rdf/v1#Geographic). Such 'RWO' resources *do exist* - for example see [Quinault Tribe of the Quinault Reservation, Washington](http://id.loc.gov/rwo/agents/n83206994), which is typed as both a madsrdf:RWO and a madsrdf:Geographic\*:

> Label: Geographic Authority
> Comment: Describes a resource whose label represents a geographic place or feature, especially when a more precise geographic determination (City, Country, Region, etc.) cannot be made.

Because existing\*\* Library of Congress RWO lookups (given above) are only available for persons, organizations, and families, no lookup can be used to retrieve those RWO resources which are *also* typed as madsrdf:Geographic. But, again, such resources do exist.

**`madsrdf:RWO` + `madsrdf:Geographic` resources are available, although not accessible via a Sinopia QA lookup. We know that the Sinopia lookup service is limited. I don't think the lack of lookup functionality for a given LC RWO resource is sufficient reason to abandon our preference for the LC RWO resources over the LC Names resources for relationships such as those being discussed here, a preference which it seems to me has been well-supported in other related discussions.**

Adam also requested an LC subject lookup for 'has related corporate body'. RDF modelling for LC subjects needs further analysis before adding as lookup to create relationships to persons, etc. I imagine LC subjects are modeled as concepts and may not meet needs for creating such relationships.

I also think additional evaluation of non-LC lookup sources is required. For example, resources in Getty ULAN are typed as http://vocab.getty.edu/ontology#PersonConcept \*\*\*.

## Notes
\* Setting aside for a moment that this multi-typing -- unlike the assignment of madsrdf:RWO + bf:Person + foaf:Person, for example -- seems somewhat contradictory, and preferring a resource typed as RWO over one typed as Authority     
\*\* Lookups available in [authorityConfig.json](https://github.com/LD4P/sinopia_editor/blob/main/static/authorityConfig.json) as of 2023-08-24   
\*\*\* Getty ULAN snippet
```
<http://vocab.getty.edu/ulan/500011104> a <http://vocab.getty.edu/ontology#PersonConcept> ;
        [...]
```
ISNI lookups are also configured for PTs creating relationships to persons. Judging by the [published data model](https://isni.org/page/linked-data/); seems ISNI resources are typed as either schema:Person or schema:Organization.

**Comment by briesenberg07** [comment]
@AdamSchiff thank you for providing additional information here! Following up on some of your comments:

> When a corporate body changes its name, separate name authorities are created and linked together reciprocally.

You provided some MARC Authority snippets to illustrate (excerpted):
```
010  n  99804490 
1102 University of Washington. ǂb Department of Anesthesiology
5102 ǂw r ǂi Successor: ǂa University of Washington. ǂb Department of Anesthesiology & Pain Medicine
```
```
010  no2020097621 
1102 University of Washington. ǂb Department of Anesthesiology & Pain Medicine
5102 ǂw r ǂi Predecessor: ǂa University of Washington. ǂb Department of Anesthesiology
```

> We will use the name for the creator appropriate to the time period when the work was created.  So a law promulgated by Ceylon will have 110 1 Ceylon, $e enacting jurisdiction, but a law after May 1972 will have 110 1 Sri Lanka, $e enacting jurisdiction. 

Above I argue *against* using what I'm calling 'LC Names resources' and *for* using what I'm calling 'LC RWO resources' based on the way each are modeled. 

But what about from the perspective of the successor/predecessor relationships which you'd like to express in some way in our RDF data? Do either of these types of RDF resources model successor/predecessor relationships?

**LC Names resources**
```
<http://id.loc.gov/authorities/names/n99804490> a madsrdf:Authority, # Dept of Anesthesiology
    madsrdf:CorporateName,
    skos:Concept ;
    [...]
    madsrdf:hasRelatedAuthority <http://id.loc.gov/authorities/names/no2020097621> ; # Anesthesiology & Pain Medicine
    [...]
    madsrdf:identifiesRWO <http://id.loc.gov/rwo/agents/n99804490> ; 
    [...]
```
```
<http://id.loc.gov/authorities/names/no2020097621> a madsrdf:Authority, # Anesthesiology & Pain Medicine
    madsrdf:CorporateName,
    skos:Concept ;
    [...]
    madsrdf:hasRelatedAuthority <http://id.loc.gov/authorities/names/n99804490> ; # Dept of Anesthesiology
    [...]
    madsrdf:identifiesRWO <http://id.loc.gov/rwo/agents/no2020097621> ;
    [...]
```
**LC RWO resources**
```
<http://id.loc.gov/rwo/agents/n99804490> a <http://id.loc.gov/ontologies/bibframe/Organization>,
    madsrdf:RWO,
    foaf:Organization ;
    [...] # no madsrdf:hasRelatedAuthority
    madsrdf:isIdentifiedByAuthority <http://id.loc.gov/authorities/names/n99804490> .
```
```
<http://id.loc.gov/rwo/agents/no2020097621> a <http://id.loc.gov/ontologies/bibframe/Organization>,
    madsrdf:RWO,
    foaf:Organization ;
    [...] # no madsrdf:hasRelatedAuthority
    madsrdf:isIdentifiedByAuthority <http://id.loc.gov/authorities/names/no2020097621> .
```
**For these examples:**

1. Relation between predecessor and sucessor is generalized to 'has related authority'.
2. This relation is present in the triples for 'Names resources,' but not for 'RWO resources.'

Note that for the two examples above, description sets for Name resources = ~140 lines (including, as has been remarked on elsewhere, a reproduction of the corresponding RWO resource description), while those for RWO resources = ~30 lines.

Descriptions for these two example RWO resources are less rich and missing relationships which we think are important. Other *kinds* of RWO resources (persons, orgs, families) are likely the same?

**NOTE that** at present we still use the available QA lookups for RWO (not names) resources in many places, and that the suggestion to add the LC geographic Names (not RWO) lookup was initially only made because no lookup was available for geographic RWOs. Based on previous discussions I believed that when creating relationships to things like people, corporate bodies, and places, we preferred resources which are *not* typed as madsrdf:Authority (such as, but not limited to, what I call 'LC Name resources') as a general rule. 

❓ Does the above information change this preference, if such a preference existed?

**Comment by AdamSchiff** [comment]
This is quite interesting to see (that LC has dumbed down the relationships in id.loc.gov, which doesn't make sense to me when we explicitly code the nature of the relationship with RDA elements).  Here are some agent relationships from the LC/NACO file:
```
010  no2019112479
040  NL-UtRU ǂb eng ǂe rda ǂc NL-UtRU
046  ǂf 1917-12-20 ǂg 1983-01-12 ǂ2 edtf
1001 Aalberse, Han B., ǂd 1917-1983
370  Bruinisse (Netherlands) ǂb Bruinisse (Netherlands)
374  Publishers and publishing ǂa Novelists ǂa Translators ǂ2 lcsh
5001 ǂw r ǂi Real identity: ǂa Keulen, Johan van, ǂd 1917-1983
```
```
010  no2019112558
040  NL-UtRU ǂb eng ǂe rda ǂc NL-UtRU
046  ǂf 1917-12-20 ǂg 1983-01-12 ǂ2 edtf
1001 Keulen, Johan van, ǂd 1917-1983
370  Bruinisse (Netherlands) ǂb Bruinisse (Netherlands)
374  Publishers and publishing ǂa Novelists ǂa Translators ǂ2 lcsh
4001 Keulen, Johannes van, ǂd 1917-1983
5001 ǂw r ǂi Alternate identity: ǂa Aalberse, Han B., ǂd 1917-1983
```
```
010  n  87128107
040  DNAL ǂb eng ǂe rda ǂc DNAL ǂd DNAL ǂd DLC ǂd WaU
0247 141986822 ǂ2 viaf ǂ1 http://www.viaf.org/viaf/141986822
1112 AFIA Nutrition Symposium
368  Symposium
368  Congresses and conventions ǂ2 lcsh ǂ0 http://id.loc.gov/authorities/subjects/sh85031115
372  Animal nutrition ǂ2 lcsh ǂ0 http://id.loc.gov/authorities/subjects/sh85005216
372  Animal feeding ǂ2 lcsh ǂ0 http://id.loc.gov/authorities/subjects/sh87007869
373  American Feed Industry Association ǂ2 naf ǂ0 http://id.loc.gov/authorities/names/n86131110 ǂ1 http://id.loc.gov/rwo/agents/n86131110
377  eng ǂ0 http://id.loc.gov/vocabulary/languages/eng
4112 American Feed Industry Association Nutrition Symposium
4112 Nutrition Symposium, AFIA
4112 Nutrition Symposium (American Feed Industry Association)
5102 ǂw r ǂi Sponsoring corporate body: ǂa American Feed Industry Association ǂ4 http://rdaregistry.info/Elements/u/P60676 ǂ0 http://id.loc.gov/authorities/names/n86131110 ǂ1 http://id.loc.gov/rwo/agents/n86131110
```
```
010  no2021046227
040  NNU ǂb eng ǂe rda ǂc NNU
0247 0000000446480070 ǂ2 isni ǂ1 http://isni.org/isni/0000000446480070
0247 Q5476713 ǂ2 viaf ǂ1 http://viaf.org/viaf/Q5476713
046  ǂs 2013-06-28 ǂt 2019-03-20 ǂ2 edtf
1102 21st Century Fox (Firm)
370  ǂe New York (N.Y.) ǂ2 naf
372  Television production companies ǂa Motion picture studios ǂa Press ǂa Mass media ǂa Broadcast journalism ǂ2 lcsh
377  eng
4102 21CF (Firm)
5001 ǂw r ǂi Founder: ǂa Murdoch, Rupert, ǂd 1931-
5102 ǂw r ǂi Predecessor of split: ǂa News Corporation
5102 ǂw r ǂi Hierarchical subordinate: ǂa Twentieth Century-Fox Film Corporation
5102 ǂw r ǂi Absorbing corporate body: ǂa Walt Disney Company
```
```
010  no2012154179
040  MsU ǂb eng ǂe rda ǂc MsU ǂd MsU ǂd DLC ǂd UPB ǂd WaU ǂd IaU ǂd WaU
0247 294008977 ǂ2 viaf ǂ1 http://www.viaf.org/viaf/294008977
1003 Baroni (Family : ǂc Natchez, Miss.)
370  ǂc United States ǂ2 naf ǂ0 http://id.loc.gov/authorities/names/n78095330 ǂ1 http://id.loc.gov/rwo/agents/n78095330
370  ǂe Natchez (Miss.) ǂ2 naf ǂ0 http://id.loc.gov/authorities/names/n79042247 ǂ1 http://id.loc.gov/rwo/agents/n79042247
376  Families ǂ2 lcdgt ǂ0 http://id.loc.gov/authorities/demographicTerms/dg2015060157
376  ǂb Baroni, Marjorie Rushing, 1924-1986 ǂ2 naf ǂ0 http://id.loc.gov/authorities/names/no2012154119 ǂ1 http://id.loc.gov/rwo/agents/no2012154119
5001 ǂw r ǂi Progenitor: ǂa Baroni, Marjorie Rushing, ǂd 1924-1986 ǂ4 http://rdaregistry.info/Elements/u/P60683 ǂ0 http://id.loc.gov/authorities/names/no2012154119 ǂ1 http://id.loc.gov/rwo/agents/no2012154119
```
```
010  no2012154119 ǂz n 2007164560
040  MsU ǂb eng ǂc MsU ǂe rda ǂd DLC ǂd MsU ǂd UPB ǂd DLC ǂd UPB ǂd DLC
046  ǂf 1924-08-16 ǂg 1986-03-03 ǂ2 edtf
1001 Baroni, Marjorie Rushing, ǂd 1924-1986
370  Mississippi ǂc United States ǂf Natchez (Miss.) ǂ2 naf
372  Women--Religious life--Mississippi ǂa Civil rights movements--Mississippi ǂa Mississippi--Race relations ǂ2 lcsh
373  Copiah-Lincoln Junior College
373  Mississippi Council on Human Relations ǂa University of Southern Mississippi ǂ2 naf
374  African American women civil rights workers ǂa Newspaper editors ǂ2 lcsh
377  eng
4001 Baroni, Marge Rushing, ǂd 1924-1986
4001 Baroni, Marge, ǂd 1924-1986
4001 Rushing, Marjorie, ǂd 1924-1986
5003 ǂw r ǂi Descendants: ǂa Baroni (Family : ǂc Natchez, Miss.)
```
Are these all generalized to "related authority" too?

Regarding your question, ❓ Does the above information change this preference, if such a preference existed?, I guess we need to have a good discussion about this.  All I know is that we need to be able to express a relationship as creator or contributor to works and expressions for geographic entities that are also jurisdictions, and these entities reside in the LC/NACO Authority File tagged as geographic names.  Interestingly, the MARC 21 Authority Format just refers to them as "geographic names":

> **[151 - Heading-Geographic Name (NR)](https://www.loc.gov/marc/authority/ad151.html)**
> Field Definition and Scope:
> Geographic name used as a heading in an established heading record, an established heading and subdivision record, a traced or an untraced reference record, or a reference and subdivision record.
> In an established heading record, field 151 contains the established form of a geographic name. In records for references, this field contains an unestablished form of a geographic name.

> X51 - Geographic Names-General Information
> Guidelines:
> Jurisdiction names (including those that are ecclesiastical entities) alone or followed by subject subdivisions are X51 geographic names.

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
(206) 543-8409
***@***.***

**Reply by briesenberg07** [reply]
Thank you @AdamSchiff for sending more test data! I would like to query the RDF representation of these authorities when I'm back so we can be more confident that this is in fact the pattern.  

to do @briesenberg07 - query a test set including Adam's records above along the lines of [previous queries on LC resources](https://gist.github.com/briesenberg07/127532b2b1088141a39b8d5901205acd#file-get_and_query-ipynb)
